### ISSUE TITLE
Add backquote support

### ISSUE THREAD

**Issue by Kudo** [i18next/i18next-scanner#57]
#### Version
- i18next: <i>10.0.7</i>
- i18next-scanner: <i>2.1.2</i>

#### Configuration
```js
{ // i18next-scanner options
    defaultValue: '__STRING_NOT_TRANSLATED__',
    resource: {
      loadPath: 'i18n/{{lng}}/{{ns}}.json',
      savePath: 'i18n/{{lng}}/{{ns}}.json',
      jsonIndent: 2
    },
    nsSeparator: false,
    keySeparator: false
}
```

For multiline string, I sometimes use ES6 template strings and it seems i18next-scanner does not support backquote according to the [test case](https://github.com/i18next/i18next-scanner/blob/master/test/fixtures/quotes.js).

My test case is:
```js
i18next.t(`aString`);
```
or even further

```js
i18next.t(`line1
  line2
  line3`);
```

Could you please help to add the backquote parsing support?
Thank you.

**Comment by cheton** [comment]
The assumption for using template literals is to embed expressions within a string, however, it's not possible for i18next-scanner to find the translation key using static analysis.

I can add support for matching back-tick (` `) character, but it would need user's help to specify an option like `defaultKey`, as either below:

* defaultKeys as an array of translation keys
```js
i18next.t(`added ${something}`, {
    defaultKeys: [
        'added foo',
        'added bar'
    ]
});
```

* defaultKey as an object containing all the variables
```js
i18next.t(`added ${something}`, {
    defaultKey: {
        'something': ['foo', 'bar']
    }
});
```

**Comment by Kudo** [comment]
The interpolation mechanism in i18next works well.
My case is just to use template strings for multiline strings and we don't need the replace the content.
I.e. I will do something like that from your case.

```js
i18next.t(`added {{something}}
  and {{bar}}`, { something: 'foo', bar: 'bar' });
```

Well, adding this feature may confuse new user especially between the partial template strings support and interpolation.
Please consider and if there are any concerns just forget me and I will close this issue.

**Comment by cheton** [comment]
Sure. I will update the parser to support back-tick (\` \`) characters, just to let you know the translation key will have newline characters like below:

```json
{
    "added {{something}}\n and {{bar}}": ""
}
```

**Comment by cheton** [comment]
Published i18next-scanner@2.1.3

**Comment by Kudo** [comment]
It works like a charm. Thank you so much for quick enhancement.
